deploy/releases/1.2.0/kubectl/readme.adoc

```diff
@@ -0,0 +1,75 @@
+
+== Installation
+
+The Websphere Liberty Operator can be installed to:
+
+* watch own namespace
+* watch another namespace
+* watch all namespaces in the cluster
+
+Appropriate roles and bindings are required to watch another namespace or watch all namespaces.
+
+---
+
+. Install Custom Resource Definitions (CRDs) for `WebsphereLibertyApplication` and day-2 operations `WebsphereLibertyTrace` and `WebsphereLibertyDump`. This needs to be done only ONCE per cluster:
++
+[source,sh]
+----
+    kubectl apply -f https://raw.githubusercontent.com/WASdev/websphere-liberty-operator/main/deploy/releases/1.2.0/kubectl/websphereliberty-app-crd.yaml
+----
+
+. Install the `Websphere Liberty Operator:
```

Comment:
Needs another ` at the end

Reply:
Good catch. I removed ` since it's not needed when referring to the Operator